ci-operator/templates/openshift/openshift-azure/build-base-image.yaml

```diff
@@ -0,0 +1,224 @@
+        cp /usr/share/ansible/openshift-ansible/inventory/dynamic/injected/secret /tmp/shared/credentials
+        sed -i -e "s#TENANT_ID#TENANT#" /tmp/shared/credentials
+        sed -i -e "s#CLIENT_SECRET#SECRET#" /tmp/shared/credentials
+        source /tmp/shared/credentials
+
+        ansible-playbook \
+          -e openshift_azure_resource_group_name=$RESOURCE_GROUP \
+          -e openshift_azure_resource_location=eastus \
+          -e openshift_azure_input_image_ns=images \
+          -e openshift_azure_input_image_prefix=${DEPLOY_OS}-root \
+          -e openshift_azure_output_image_ns=images \
+          -e openshift_azure_output_image_name=$IMAGE_NAME \
+          -e openshift_azure_vm_size=$AZURE_VM_SIZE \
+          playbooks/azure/openshift-cluster/build_base_image.yml
+
+        touch /tmp/shared/image_create
+
+    - name: tagimage
+      image: ${IMAGE_ANSIBLE}
+      resources:
+        requests:
+          cpu: 1
+          memory: 300Mi
+        limits:
+          cpu: 3
+          memory: 4Gi
+      volumeMounts:
+      - name: shared-tmp
+        mountPath: /tmp/shared
+      - name: cluster-secrets-azure
+        mountPath: /etc/azure/credentials
+      env:
+      - name: HOME
+        value: /usr/share/ansible/openshift-ansible
+      command:
+      - /usr/local/bin/entrypoint-provider
+      args:
+      - /bin/bash
+      - -c
+      - |
+        #!/bin/bash
+        # error handling and sync code
+        set -euo pipefail
+
+        trap '[ "$?" == 0 ] || touch /tmp/shared/exit' EXIT
+        trap 'kill $(jobs -p); exit 0' TERM
+
+        # wait until testing our image has succeeded
+        while true; do
+          if [[ -f  /tmp/shared/exit ]]; then
+             exit 1
+          fi
+          sleep 15 & wait
+          continue
+        done
+
+        # Get resource group and image name
+        source /tmp/shared/environment
+
+        # Source azure creds
+        source /etc/azure/credentials/secret
+        az login --service-principal -u ${AZURE_CLIENT_ID} -p ${AZURE_CLIENT_SECRET} --tenant ${AZURE_TENANT_ID} &>/dev/null
+
+        ansible-playbook \
+          -e openshift_azure_input_image_ns=images \
+          -e openshift_azure_input_image_name=$IMAGE_NAME \
+          playbooks/azure/openshift-cluster/tag_image_as_valid.yml
+
+        touch /tmp/shared/exit
+
+    # Performs cleanup of all created resources
+    - name: teardown
+      image: registry.svc.ci.openshift.org/azure/test-base:latest
+      volumeMounts:
+      - name: artifacts
+        mountPath: /tmp/artifacts
+      - name: shared-tmp
+        mountPath: /tmp/shared
+      - name: openshift-azure
+        mountPath: /go
+      - name: cluster-secrets-azure
+        mountPath: /etc/azure/credentials
+      env:
+      - name: HOME
+        value: /tmp/shared/home
+      command:
+      - /bin/bash
+      - -c
+      - |
+        #!/bin/bash
+        function teardown() {
+          set +e
+
```

Comment:
You need to az group delete here.

Reply:
alright

Reply:
I also needed to perform an `az login` here